cmd/tempo-cli/cmd-list-block.go

```diff
@@ -85,13 +100,16 @@ func dumpBlock(r tempodb_backend.Reader, c tempodb_backend.Compactor, tenantID s
 		maxObjSize := 0
 		minObjSize := 0
 
+		allKVP := kvPairs{}
 		printStats := func() {
 			fmt.Println()
 			fmt.Println("Scanning results:")
 			fmt.Println("Objects scanned : ", i)
 			fmt.Println("Duplicates      : ", dupe)
 			fmt.Println("Smallest object : ", humanize.Bytes(uint64(minObjSize)))
 			fmt.Println("Largest object  : ", humanize.Bytes(uint64(maxObjSize)))
```

Comment:
While you're in here - can we print the ID for the largest object? Currently trying to debug some OOMing compactors and it would help to know if they are trying to combine a super large trace across multiple blocks

Reply:
done. good call.